src/render.js

```diff
@@ -13,8 +13,12 @@ import options from './options';
  */
 export function render(vnode, parentDom, replaceNode) {
 	if (options._root) options._root(vnode, parentDom);
-	let oldVNode = replaceNode && replaceNode._children || parentDom._children;
 
+	let oldVNode = replaceNode && replaceNode._children || parentDom._children;
+	let isHydrating = false;
```

Comment:
Maybe leaving this out will save some bytes as in let `let isHydrating = replaceNode === null`

Reply:
This cut 7B